enclave/core/sgx/calls.c

```diff
@@ -168,6 +169,19 @@ static oe_result_t _handle_init_enclave(uint64_t arg_in)
 
             oe_enclave = enclave;
 
+            if (__oe_get_enclave_create_zero_base_flag())
```

Comment:
A better place for this check would be in this file: https://github.com/openenclave/openenclave/blob/cd72fd7069488ba6f453c8f5f47bd9fd9a6e6c0d/enclave/core/sgx/init.c#L110

Other constraints about enclave layout are asserted here.

Reply:
moved check to init.c